src/test/java/org/broadinstitute/hellbender/tools/spark/sv/NovelAdjacencyReferenceLocationsUnitTest.java

```diff
@@ -158,7 +158,7 @@ public void testGetAssembledBreakpointsFromAlignmentRegionsWithOverlappingAlignm
         Assert.assertEquals(chimericAlignment.regionWithLowerCoordOnContig, region1);
         Assert.assertEquals(chimericAlignment.regionWithHigherCoordOnContig, region3);
         Assert.assertEquals(chimericAlignment.insertionMappings.size(), 1);
-        Assert.assertEquals(chimericAlignment.insertionMappings.get(0), "1-contig-1:484-525:20,23103196,-,483S42M968S,60,2");
+        Assert.assertEquals(chimericAlignment.insertionMappings.get(0), "1_contig-1_484_525_20_23103196_-_483S42M968S_60_2");
```

Comment:
Perhaps you should construct the "actual" string using the constant PACKED_STRING_REP_SEPARATOR so that this test don't fail if you happen to change that constant value.